test-data/unit/semanal-errors.test

```diff
@@ -1330,3 +1330,13 @@ a = ('spam', 'spam', 'eggs', 'spam')  # type: Tuple[str]
 [out]
 main:3: error: Name 'a' already defined
 main:4: error: Name 'a' already defined
+
+[case testUnknownContextDuplicateDef]
+from m import A
+class A:
+    pass
+[file m.py]
+class A:
+    pass
+[out]
+main:2: error: Name 'A' already defined (line unavailable; possibly an import)
```

Comment:
If a module is simply imported twice, there's no error message. Did you mean some other test?